spec/round_spec.rb

```diff
@@ -0,0 +1,129 @@
+require 'rspec'
+require './lib/card'
+require './lib/deck'
+require './lib/turn'
+require './lib/round'
+
+describe 'Round' do
+  before :each do
+    @card_1 = Card.new("What is the capital of Alaska?", "Juneau", :Geography)
+    @card_2 = Card.new("The Viking spacecraft sent back to Earth photographs and reports about the surface of which planet?", "Mars", :STEM)
+    @card_3 = Card.new("Describe in words the exact direction that is 697.5° clockwise from due north?", "North north west", :STEM)
+    @deck = Deck.new([@card_1, @card_2, @card_3])
+    @round = Round.new(@deck)
+  end
+
+  it 'will have an instance of a Round' do
+
+    expect(@round).to be_instance_of Round
+  end
+
+  it 'will be able to read the deck variable' do
+
+    expect(@round.deck).to eq @deck
+  end
+
+  it 'will be able to take turns' do
+    @round.turns
+
+    expect(@round.turns).to eq []
```

Comment:
this happens a lot in your tests -- you do not need to run a method in order to assert it on your expectation line. AKA, line 27 is unnecessary.